docs/reference/release-notes/7.11.asciidoc

```diff
@@ -288,6 +296,9 @@ Machine Learning::
 * Fix edge case for data frame analytics where a field mapped as a keyword actually has boolean and string values in the `_source` {es-pull}64826[#64826]
 * Fix job ID in C++ logs for normalize and memory estimation {es-pull}63874[#63874] (issues: {es-issue}54636[#54636], {es-issue}60395[#60395])
 * Truncate long audit messages {es-pull}64849[#64849] (issue: {es-issue}64570[#64570])
+* Fix potential cause for log errors from CXMeansOnline1d {ml-pull}1586[#1586]
+* Fix scaling of some hyperparameters for Bayesian optimization {ml-pull}1612[#1612]
+* Fix missing state in persist and restore for anomaly detection. This caused suboptimal modelling after a job was closed and reopened or failed over to a different node {ml-pull}1668[#1668]
```

Comment:
```suggestion
* Fix missing state in persist and restore for anomaly detection. This caused suboptimal modeling after a job was closed and reopened or failed over to a different node {ml-pull}1668[#1668]
```

My (UK) browser is showing "modeling" as a typo, but is it the preferred US spelling that we should use publicly?

Reply:
I don't mind having some international flavour in our docs sometimes, but we can synch on US spelling here.